OpenRA.Game/Player.cs

```diff
@@ -130,20 +130,27 @@ public Player(World world, Session.Client client, PlayerReference pr)
 			fogVisibilities = PlayerActor.TraitsImplementing<IFogVisibilityModifier>().ToArray();
 
 			// Enable the bot logic on the host
-			IsBot = botType != null;
-			if (IsBot && Game.IsHost)
+			ActivateBot(botType);
+
+			stanceColors.Self = ChromeMetrics.Get<Color>("PlayerStanceColorSelf");
+			stanceColors.Allies = ChromeMetrics.Get<Color>("PlayerStanceColorAllies");
+			stanceColors.Enemies = ChromeMetrics.Get<Color>("PlayerStanceColorEnemies");
+			stanceColors.Neutrals = ChromeMetrics.Get<Color>("PlayerStanceColorNeutrals");
+		}
+
+		public void ActivateBot(string botType)
+		{
+			if (!string.IsNullOrEmpty(botType) && Game.IsHost)
 			{
 				var logic = PlayerActor.TraitsImplementing<IBot>().FirstOrDefault(b => b.Info.Name == botType);
 				if (logic == null)
 					Log.Write("debug", "Invalid bot type: {0}", botType);
 				else
+				{
 					logic.Activate(this);
+					Log.Write("debug", "Activated bot {0}".F(botType));
```

Comment:
Ah, nevermind.  Your comment on the last thread relating to this made me think that you didn't want to change it, so I didn't catch that you moved from `Game.Debug` to `Log.Write`.